src/data/MetaDataItem.py

```diff
@@ -23,8 +26,11 @@ def to_json(self) -> dict:
 
     def to_json_str(self) -> str:
         return json.dumps(self.to_json(), sort_keys=True, indent=2)
-    
+
     def to_file(self):
         # Write the output to disk
         with open(self.id + '_metadata.json', 'w') as outfile:
-            json.dump(self.to_json, outfile, sort_keys=True, indent=2)
\ No newline at end of file
+            json.dump(self.to_json, outfile, sort_keys=True, indent=2)
+
+item = MetaDataItem(1,"a", "url1", "abc", "desc", "loc")
```

Comment:
Is this for testing? If so it should be removed.

Reply:
yep, good catch